x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning.tsx

```diff
@@ -82,10 +84,12 @@ export const RelevanceTuning: React.FC<Props> = ({ engineBreadcrumb }) => {
 
     return (
       <EuiFlexGroup>
```

Comment:
@daveyholler not sure if you want to weigh in on this one, but if we're following the old standalone UI it looks like this shouldn't be stretching the preview panel to match the left column height:

```suggestion
      <EuiFlexGroup alignItems="flexStart">
```

Before:
<img width="1135" alt="" src="https://user-images.githubusercontent.com/549407/109746195-dccb9e80-7b89-11eb-8ae1-c19b533bcb40.png">
...
<img width="1120" alt="" src="https://user-images.githubusercontent.com/549407/109746200-dfc68f00-7b89-11eb-989b-34e9322c0100.png">


After:
<img width="1126" alt="" src="https://user-images.githubusercontent.com/549407/109746154-c7ef0b00-7b89-11eb-868d-5b083cab4318.png">

Reply:
http://localhost:5601/app/enterprise_search/app_search/engines/a/relevance_tuning